spec/services/manifest_builder/by_resource_type/manifest_builder_scanned_resource_spec.rb

```diff
@@ -158,9 +158,9 @@ def logical_structure(file_set_id)
     let(:scanned_resource) do
       FactoryBot.create_for_repository(:scanned_resource, title: ["title1", "title2"])
     end
-    it "uses an array" do
+    it "displays a root label with a single string as label is a single value in Prezi V2" do
       output = manifest_builder.build
-      expect(output["label"]).to eq ["title1", "title2"]
+      expect(output["label"]).to eq "title1"
```

Comment:
Same question as above - what's the impact of this change? Did UV used to concat?

Reply:
UV does not concat as far as I can tell.